backend/api/src/main/kotlin/de/keepmealive3d/event/LiveDataEventHandler.kt

```diff
@@ -1,7 +1,43 @@
 package de.keepmealive3d.event
 
-class LiveDataEventHandler: EventHandler {
+import com.influxdb.client.domain.WritePrecision
+import com.influxdb.client.kotlin.InfluxDBClientKotlinFactory
+import com.influxdb.client.write.Point
+import de.keepmealive3d.config.Config
+import kotlinx.coroutines.runBlocking
+import java.time.Instant
+
+class LiveDataEventHandler(conf: Config) : EventHandler {
+    private val client = InfluxDBClientKotlinFactory.create(
+        "http://${conf.databases.influx.host}:${conf.databases.influx.port}",
+        conf.databases.influx.token.toCharArray(),
+        conf.databases.influx.org,
+        conf.databases.influx.bucket
+    )
+
+
     fun receive(topic: String, payload: String) {
+
+        runBlocking {
```

Comment:
usually a bad idea to hide a `runBlocking`  within a function and not telling the caller about it. It may be an option to mark the receive function as suspend able and let the caller handle it, e.g. if he wants to wait using a `runBlocking`. Another option may be to actually launch a own coroutine with `launch{}`.

Reply:
Done